exercicioLogica18/src/main/java/org/example/MotoPartidaPedal.java

```diff
@@ -0,0 +1,22 @@
+package org.example;
+
+public class MotoPartidaPedal extends Moto {
+    public MotoPartidaPedal(String marca, String modelo, int velocidadeMaxima, String tipo) {
+        super(marca, modelo, velocidadeMaxima, tipo);
+
+        setRodas(2);
+    }
+
+    public boolean condicaoLigarMoto (boolean acelerar) {
```

Comment:
Moto já possui método ligarMoto, poderia se utilizar de sobrescrita e somente alterar o método para ligar quando a condição necessária for satisfeita. Classe Automovel também já possui propriedade "acelerar", acho que não seria necessário passar uma nova propriedade "acelerar" por parâmetro.